src/js/media-controller.js

```diff
@@ -239,7 +237,9 @@ class MediaController extends MediaContainer {
     try {
       const volPref = window.localStorage.getItem('media-chrome-pref-volume');
       if (volPref !== null) media.volume = volPref;
-    } catch (e) { }
+    } catch (e) {
+      console.error('Error getting volume pref', e);
```

Comment:
Tracking the last volume that was set is a bonus, not a required feature, so we don't need to throw an error here. This would only fail if the browser doesn't support localStorage or its disabled, which is fine. At most I think we'd throw a `console.debug()` or similar.

Reply:
cool,  I'll use `console.debug`